dags/taar_weekly.py

```diff
@@ -14,14 +17,28 @@
 taar_ensemble_cluster_name = "dataproc-taar-ensemble"
 taar_gcpdataproc_conn_id = "google_cloud_airflow_dataproc"
 
+TAAR_BIGTABLE_INSTANCE_ID = Variable.get("taar_bigtable_instance_id")
+TAAR_ETL_STORAGE_BUCKET = Variable.get("taar_etl_storage_bucket")
+TAAR_PROFILE_PROJECT_ID = Variable.get("taar_gcp_project_id")
+TAAR_DATAFLOW_SUBNETWORK = Variable.get("taar_dataflow_subnetwork")
+
+TAAR_ETL_CONTAINER_IMAGE = (
+    "gcr.io/moz-fx-data-airflow-prod-88e0/taar_gcp_etl:0.4.7"
```

Comment:
add ref comment on where the repo that builds this image is

Reply:
Added in 3600e88e7a54